pkgs/development/tools/gtk-mac-bundler/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "gtk-mac-bundler-${version}";
+  version = "0.7.4";
+
+  src = fetchFromGitHub {
+    owner = "GNOME";
+    repo = "gtk-mac-bundler";
+    rev = "bundler-${version}";
+    sha256 = "1kyyq2hc217i5vhbfff0ldgv0r3aziwryd1xlck5cw3s6hgskbza";
+  };
+
+  installPhase = ''
+  	mkdir -p $out/bin
+    sed "s,@PATH@,$out/share,g" < gtk-mac-bundler.in > $out/bin/gtk-mac-bundler
```

Comment:
You can replace variables like that with [substitute](http://nixos.org/nixpkgs/manual/#ssec-stdenv-functions), perhaps that's a little bit nicer.

```
substitute gtk-mac-bundler.in $out/bin/gtk-mac-bundler \
  --subst-var-by PATH $out/share
```